compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/ContextMenuProvider.desktop.kt

```diff
@@ -52,10 +52,39 @@ fun ContextMenuArea(
     state: ContextMenuState = remember { ContextMenuState() },
     enabled: Boolean = true,
     content: @Composable () -> Unit
+) {
+    ContextMenuArea(
+        items = items,
+        state = state,
+        modifier = Modifier.contextMenuOpenDetector(state, enabled),
+        content = content
+    )
+}
+
+/**
+ * Defines a container where context menu is available. Representation of menu is defined by
+ * [LocalContextMenuRepresentation].
+ *
+ * This overload does not trigger the opening of the context menu; it's up to the caller to do so
+ * by passing a [modifier] that does so.
```

Comment:
It will be confusing, we can avoid it by default args

Reply:
I don't think it's confusing, it's just not very convenient, but since this API is for advanced developers, I think it's better to prefer clarity (who's responsible for what, in this case) over convenience.